packages/headless/src/controllers/core/folded-result-list/headless-core-folded-result-list.test.ts

```diff
@@ -0,0 +1,145 @@
+import {search, configuration, folding, query} from '../../../app/reducers';
```

Comment:
You'll see the `reducers` file has been deleted in master as of today.

This is part of an initiative to reduce bundle size for our clients. You will need to imports reducers from their respective files instead 

ie: 
```typescript
import {searchReducer as search} from '../../../features/search/search-slice'`
// etc.
```